public/locales/en.json

```diff
@@ -144,6 +144,7 @@
   "migrateToGauge": "{{farmName}} Farm upgrade! Old farm is deprecated. You are no longer getting staking rewards. Please upgrade to the gauge farm.",
   "migrationAmount": "Migration Amount",
   "migrationExplain": "This pool is outdated. We recommend migrating to the new gas optimized version to earn better returns.",
+  "migrationGaugeWarningMsg": "Farm upgrade: LP rewards have migrated to the gauge. To automatically claim your remaining rewards, go to the Deposit or Withdraw page and click the button in the top banner.",
```

Comment:
This gets the right gist, but would you mind sharing this in a slack channel so the markops folks can help us workshop it a little? We'll probably also want to include a link to a sections of the docs that explains minichef vs Gauges in greater detail.